libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilotImpl.kt

```diff
@@ -109,15 +110,20 @@ internal class MapboxCopilotImpl(
     }
     private var initRoute = false
     private lateinit var routesObserver: RoutesObserver
+    private val isMapboxNavigationAppSetup = MapboxNavigationApp.isSetup()
 
     /**
      * start
      */
     fun start() {
         registerUserFeedbackCallback(userFeedbackCallback)
-        MapboxNavigationApp.lifecycleOwner.lifecycle.addObserver(
-            foregroundBackgroundLifecycleObserver
-        )
+        if (isMapboxNavigationAppSetup) {
+            MapboxNavigationApp.lifecycleOwner.lifecycle.addObserver(
+                foregroundBackgroundLifecycleObserver
+            )
+        } else {
+            ProcessLifecycleOwner.get().lifecycle.addObserver(foregroundBackgroundLifecycleObserver)
```

Comment:
Do we need `MapboxNavigationApp.lifecycleOwner.lifecycle.addObserver` at all? Won't `ProcessLifecycleOwner` work in that case as well?

Reply:
`ProcessLifecycleOwner` doesn't support Android Auto.